compiler/test/grainfmt/includes.input.gr

```diff
@@ -11,7 +11,7 @@ include /* special include */ "array"
 include "array" as  /* special include */ Foo
 from List use { length, map, forEach as each }
 from Opt use { MutableOpt, ImmutableOpt as Imm, type Opt, type Opt as OptAlias }
-from Opt use { MutableOpt, /* comment1 */ ImmutableOpt /* comment2 */ as /* comment3 */ Imm /* comment4 */, /* comment5 */ type /* comment6 */ Opt, type Opt as /* comment7 */ OptAlias }
+from Opt use { MutableOpt, /* comment1 */ ImmutableOpt /* comment2 */ as /* comment3 */ Imm /* comment4 */, /* comment5 */ type /* comment6 */ Opt, type Opt as /* comment7 */ OptAlias, exception Exc as /* comment8 */ E }
```

Comment:
For completeness and to follow the rest it would probably make sense to add the test here without `as` as well to mirror the variable and type import.
```suggestion
from Opt use { MutableOpt, /* comment1 */ ImmutableOpt /* comment2 */ as /* comment3 */ Imm /* comment4 */, /* comment5 */ type /* comment6 */ Opt, type Opt as /* comment7 */ OptAlias, , exception /* comment8 */ Exc, exception Exc as /* comment9 */ E }
```